tests/cypress/support/accessibility.js

```diff
@@ -1,7 +1,7 @@
 const a11yOptions = {
```

Comment:
It's pointless and potentially confusing to define an a11Options object that is never used. I suggest deleting it instead if we're not going to use it anyway.

Reply:
My bad, misread the code.